### DISCUSSION TITLE
quarto not generating index.xml file for rss feed in quarto site

### DISCUSSION THREAD

**Discussion by boshek** [quarto-dev/quarto-cli#10946] [Q&A]
### Description

I have [this site](https://samalbers.science/) for which I'd like to add an rss feed. Based on the instructions [here](https://quarto.org/docs/websites/website-listings.html#feeds) I just really need to add

```
listing:
  contents: posts
  feed: true
```

to my `index.qmd` file. My understanding is then invoking `quarto render index.qmd` _should_ generate an `index.xml` file. 

However when I do [this here](https://github.com/boshek/samalbers.science/blob/c0df4b2cefcc2ef1ae7e675d77ddbe112b66a9e5/index.qmd#L9) an `index.xml` file is _not_ generated. 

At first I thought that maybe the indentation was incorrect like this [post](https://github.com/quarto-dev/quarto-cli/discussions/8682) but I don't think so. The indentation appear correct. So that I went over the quarto blog to see if a `index.xml` was in that repo: https://github.com/quarto-dev/quarto-web/tree/main/docs/blog

There was not. So now I am pretty confused. Which step am I missing to actually generated that file OR where/when does it get generated? Right now I am [not able to add](https://samalbers.science/index.xml) the RSS feed but it feels like I must be missing something.  

I am on quarto version 1.5.57 and am building the site locally and pushing it up.

**Comment by cscheid** [comment]
Unfortunately I can't compile your website with 1.5.57 after a fresh clone, so I can't help:

```
$ qv 1.5.57 render
[1/9] posts/2019-11-17-what-the-buck/index.qmd
[2/9] posts/eccc-webservice/index.qmd
[3/9] posts/a-short-history-of-me-at-rstudio-conf-2020/index.qmd
[4/9] posts/you-schema/index.qmd
[5/9] posts/hockey-age-with-ojs/index.qmd
[6/9] posts/one-small-step-for-a-package-one-giant-leap-for-a-task-view/index.qmd
[7/9] 404.qmd
pandoc: /site_libs/quarto-nav/quarto-nav.js: withBinaryFile: does not exist (No such file or directory)
```

I'm not quite sure what's causing that, but it would help if you can provide a clean example where the only issue is the feed generation.

**Reply by boshek** [reply]
Apologies. I have updated my post. Something about my 404 page was causing that so I just removed that for now and it now compiles with `quarto render`.
